openlibrary/templates/type/author/view.html

```diff
@@ -38,6 +38,15 @@
     //-->
     </script>
 
+$if page.birth_date or page.death_date:
+    $ entityTypeSchema = "https://schema.org/Person"
+$elif page.entity_type:
+    $ entityTypeSchema = "https://schema.org/Organization"
+
+$ bodyattrs = ctx.setdefault('bodyattrs', [])
+$ bodyattrs.append('itemscope')
+$ bodyattrs.append('itemtype=%s' % entityTypeSchema)
```

Comment:
Not _technically_ necessary, but good practice to have quotes around HTML attribute values.

```suggestion
$ bodyattrs.append('itemtype="%s"' % entityTypeSchema)
```

Reply:
@cdrini, really thank you for cleaning that up, appreciate it a lot, you made my day today, to be honest, I will make sure it never happens again. :) 

Actually, I did that on purpose, because the [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/u/0/) gives the error in the below picture, it is because double quotes are being rendered in the HTML body tag. I wanted to mention in the comment but I forgot to add after all that git stuff.

Should I add quotes or leave it as it is?

![Screenshot from 2020-11-18 03-30-35](https://user-images.githubusercontent.com/57295877/99456177-f6d0d080-294e-11eb-88ad-284c2c5cc508.png)
![Screenshot from 2020-11-18 03-31-10](https://user-images.githubusercontent.com/57295877/99456183-f9cbc100-294e-11eb-9361-28c0f4530692.png)